installing/installing_aws/installing-aws-outposts-remote-workers.adoc

```diff
@@ -3,6 +3,7 @@
 = Installing a cluster on AWS with remote workers on AWS Outposts
 include::_attributes/common-attributes.adoc[]
 :context: installing-aws-outposts-remote-workers
+:zone-type: AWS Outposts
```

Comment:
For the section, does it also contains the outposts remote workers and public workers mixed configuration, if it contains the configuration, we'd better add an extra limitation to `Persistent storage using AWS Elastic Block Store limitations`(line:143), 
...
_To create a Volume in the Outpost instance, the CSI driver determines the Outpost ARN based on the topology keys stored on the CSINode objects. To ensure that the CSI driver uses the correct topology values, it is necessary to use the `WaitForConsumer` volume binding mode and avoid setting allowed topologies on any new storage class created.
In addition, for the outposts remote workers and public workers mixed configuration,  the public AWS Elastic Block volume couldn't be attached to the outposts remote workers but the public AWS Elastic Block volume logic persistent volume nodeAffinity doesn't have the limitation, so better set nodeAffinity when creating workloads on public workers._
...
cc to @duanwei33

Reply:
Hi @yunjiang29 and @Phaow . Thanks for reviewing the content. I think the scope of the AWS Outposts work has grown a bit more than expecting and the content does not follow the same path as the LZ and WZ updates. I've removed all my intial AWS Outposts updates for now. 

The AWS Outposts documentation updates are best covered in a separate PR. WDYT?

^ If OK, how does my LZ and WZ content read from a QE perspective?

@makentenza @sjstout

Reply:
I'm ok using a separate PR for the Outpost. LZ and WZ content look good from storage qe perspective.